src/playlist.js

```diff
@@ -0,0 +1,66 @@
+var playItem = require('./playitem.js');
+var setupAutoadvance = require('./autoadvance.js');
+
+var playlistMaker = function(player, plist) {
+  var currentIndex = 0;
+  var autoadvanceTimeout = null;
+
+  var playlist = function playlist(list) {
+    if (typeof list === 'number') {
+      return player.playlist.item(list);
```

Comment:
Oops, that was a mistype. That said, I care more about usability of the resulting API than how/if we add more methods to the player object. I think there are a couple pieces of info we need to expose:

1. The current playlist
2. Videos in the current playlist
3. The index of the current video in the playlist

In my ideal world, we would have accessors and mutators for all of those items. In my original proposal, I had accessors and mutators for 1 & 2 by virtue of using an array-like object for `player.playlist`. I added the `item` property as an accessor/mutator for 3.

In this iteration, `player.playlist()` is an accessor/mutator for 1. I suggested adding `player.playlist(7)` as an accessor for 2 but I suppose you could use `player.playlist()[7]` as well. I think having `item` "scoped" to the playlist property makes organizational sense but `player.playlistItem(4)` doesn't seem too bad either.